uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/absence-type-model.spec.js

```diff
@@ -79,5 +86,43 @@ define([
         });
       });
     });
+
+    describe('canExpire()', function () {
+      describe('passing api parameters', function () {
+        beforeEach(function () {
+          AbsenceType.canExpire(999);
```

Comment:
I think it would be better to move this constant in a variable just after describe.